lib/syskit/roby_app/configuration.rb

```diff
@@ -121,6 +121,21 @@ class Configuration
             # likely want this
             attr_predicate :kill_all_on_process_server_connection?, true
 
+            # Indicate to the loggers which time field it should use as logical time
+            #
+            # When a type member is marked with `@meta role logical_time` Syskit will pass
+            # it for the logger as the logical time field.
+            #
+            # Right now opaque types aren't supported
```

Comment:
`This is disabled by default as opaque types aren't supported, and will silently not be logged`